src/controller/booking.controller.js

```diff
@@ -0,0 +1,109 @@
+import bookingModel from "../model/booking.model.js";
+import { response } from "express";
+
+export const bookingsGet = async (req, res = response) => {
+  const { limit, page } = req.query;
+  const query = { tp_status: "ACTIVE" };
+
+  const [total, bookings] = await Promise.all([
+    bookingModel.countDocuments(query),
+    bookingModel.find(query).skip(page * limit).limit(limit)
+  ]);
+
+  res.status(200).json({
+    total,
+    bookings,
+  });
+};
```

Comment:
muy bien, pero recuerda que `limit` y `page` al ser query params, si no estoy mal, son strings siempre, entonces, hay que convertirlos a numeros.
```suggestion
  const { limit, page } = req.query;
  const query = { tp_status: "ACTIVE" };

  const [total, bookings] = await Promise.all([
    bookingModel.countDocuments(query),
    bookingModel.find(query).skip(Number(page) * Number(limit)).limit(Number(limit))
  ]);

  res.status(200).json({
    total,
    bookings,
  });
};
```

de cualquier forma, muy bien los cambios, y de ser cierto lo que te digo, deberias validar que los query params, sean strings que se puedan convertir a numeros enteros validos, no solo si son numeros, pues seguramente fallara ese middleware